### PR TITLE
Ticket1276: Deleting all blocks also deletes all groups system test

### PR DESCRIPTION
**Pull request by simonfernandes**
ISISComputingGroup/IBEX/issues/1276

### CODE REVIEW

**Comment by AdrianPotter** [comment]
- [x] Please move the test file from the "Context" folder to the "IBEX_System_Tests" folder
- [x] Add the new test to the AllTests suite
- [ ] Check that the tests pass locally. Currently the test fails as part of the suite because the block added in "AddFirstBlock" is not removed as part of the test.
- [ ] Check the [Jenkins PR](http://epics-jenkins.isis.rl.ac.uk/job/System_Tests_build_PR/) build passes

**Comment by John-Holt-Tessella** [comment]
No longer have jenkins PR and don't need to remove block

File: RCPTT_Tests/ConfigurationTests/DeletingAllBlocksDeletesAllGroups.test [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+delete_block "8"
+delete_block "7"
+delete_block "6"
+delete_block "5"
+delete_block "4"
+delete_block "3"
+delete_block "2"
+delete_block "1"
+delete_block "0"
```
Comment (AdrianPotter) [inline RCPTT_Tests/ConfigurationTests/DeletingAllBlocksDeletesAllGroups.test]: We figured out how to do loops in RCPTT. See CurrentConfigNotInDeleteConfigDialog for example. Can we do that here please

**Review by AdrianPotter — CHANGES_REQUESTED** [review]
Very minor refactoring for loop please

**Review by AdrianPotter — APPROVED** [review]

**Pull request merged**